programs/clearing_house/src/lib.rs

```diff
@@ -146,6 +146,18 @@ pub mod clearing_house {
         )?;
 
         let spot_market_index = get_then_update_id!(state, number_of_spot_markets);
+
+        let oracle_price_data = get_oracle_price(
+            &oracle_source,
+            &ctx.accounts.oracle,
+            cast(Clock::get()?.unix_timestamp)?,
+        );
+
+        let hist_data_default = if spot_market_index == 0 {
+            HistOracleData::default()
```

Comment:
might be good for `default` to be the one with all zeros and for this function to be `deafult_quote_oracle`?